content/starting-team/roles/intro.md

```diff
@@ -2,10 +2,10 @@
 layout: intros/intro
 category: Starting a team
 title: Roles
-subtitle: Team capabilities are key to building and maintaining a user-centred service. You need team members with a range of skills and who are open to collaborating.
+subtitle: To build and maintain a user-centred service, you need team members with a range of skills and who are open to collaborating.
```

Comment:
Whoop! I missed this one we should change it to open to working together and in new ways.

There will be a few more collaborating I'll need to fix.